content/docs/dvclive/how-to/checkpoints.md

```diff
@@ -0,0 +1,42 @@
+# Integrate with DVC Checkpoints
+
+When using [DVC Checkpoints](/doc/user-guide/experiment-management/checkpoints)
+and/or enabling DVCLive's [`resume`](/doc/dvclive/api-reference/live#parameters)
+you need to add the flag
+[`persist: true`](/doc/user-guide/project-structure/pipelines-files#output-subfields)
+to all DVCLive outputs.
```

Comment:
Probably needs more motivation (why do this/ how is it useful) closer to what we have now in https://dvc.org/doc/dvclive/dvclive-with-dvc#checkpoints.

Reply:
I was trying to make this a concise how-to page. I would expect the first linked page to `user-guide/experiment-management/checkpoints` to cover the motivation for checkpoints and people coming to this page for specific guidance on proper setup, not looking for motivation on the feature